docs/archive/AddMeasurements.md

```
@@ -33,12 +33,15 @@ Oscar
 4. For each pair of height and weight measurements:
 	4.1 Calls toXML(), which converts the Measurement object to an XML representation.
 	4.2 Creates an object of type MedicalDataTransfter4, a wrapper for the XML
+	```java
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)